src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataextraction/contributions/ContributionsController.java

```diff
@@ -69,7 +69,7 @@ public void contributionsBbox(HttpServletRequest servletRequest,
         ElementsGeometry.BBOX, servletRequest, servletResponse);
     executor.extract();
   }
-
+  
```

Comment:
normally, we avoid trailing white spaces also on empty lines. :wink: 

same in line 96 below